tests/fixtures/yaml_tests/test_relative_error_margin.yaml

```diff
@@ -5,3 +5,14 @@
     salary: 2000
   output:
     income_tax: 290 # 300
+
+
+- name: "Result within variable relative error margin"
+  period: 2015-01
+  relative_error_margin:
+    default: .001
+    income_tax: 0.05
```

Comment:
So here we demonstrate that test passes because the result is within the lower and upper bounds (+-15), but should otherwise fail with the general one (+-3). Could we show that in the test to it's clear?